README.md

```diff
@@ -162,7 +162,32 @@ const lc = new LockContext()
 ```
 
 > [!NOTE]
-> At the time of this writing, the default LockContext is set to use the `sub` Identity Claim, as this is the only Identity Claim that is currently supported.
+> At the time of this writing, the default LockContext context is set to use the `sub` Identity Claim.
+
+**Custom context**
+
+If you want to use a custom context, you can pass it to the `LockContext` constructor.
+
+```typescript
+import { LockContext } from '@cipherstash/jseql/identify'
+
+// eqlClient from the previous steps
+const lc = new LockContext({
+  context: {
+    identityClaim: ['sub'],
+  },
+})
+```
+
+**Context and identity claim options**
+
+The context must be an object with an `identityClaim` property.
+The `identityClaim` property must be an array of strings that correspond to the Identity Claim(s) you want to lock the encryption operation to.
+
+| Identity Claim | Description | Default |
+| -------------- | ----------- | ------- |
+| `sub`          | The user's subject identifier. | yes |
```

Comment:
How is the default overridden? By just not including it in the `identityClaim` array? Might be good to show some more examples.